src/main/scala/no/ndla/articleimport/service/converters/RelatedContentConverter.scala

```diff
@@ -73,6 +81,29 @@ trait RelatedContentConverter {
       }
 
     }
+
+    /**
+      * @param nidsAndType
```

Comment:
kanskje en beskrivelse av disse to parameterene også, hvis vi først skal ha doc? :p